nasbench/train_seminas.py

```diff
@@ -13,13 +15,18 @@
 import torch.nn.functional as F
 import torch.backends.cudnn as cudnn
 
-import utils
-from controller import NAO
+sys.path.append('./../../libs/nasbench')
```

Comment:
This line is unnecessary if we move this functionality to `self-supervised-nas` repo.

Reply:
This line has dependency outside of this repo, hence not controllable.